### ISSUE TITLE
Changing video stream topic

### ISSUE THREAD

**Issue by jihoonl** [creativa77/rocon_web_proxy_server#52]
`concert_teleop_web` is designed to change the video topic names when robot is allocated.

The current video handler does not allow to change the video stream topic name.

https://github.com/creativa77/rocon_web_proxy_server/blob/master/main.py#L72

**Comment by pablisho** [comment]
Yes, we didn't notice that it was needed to change the topic name. 
This issue is caused by a workaround related to issue #33 which does not allow to handle multiple connections (either video or ws) per client. The workaround we made in video is to accept the first request and ignore the others.
